src/components/main/components/task/list/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button } from "../../../../collections/button";
+import { Task } from "../../../../types/task";
+import styled from "@emotion/styled";
+
+const Ul = styled.ul`
+    list-style: none;
+    display: flex;
+    justify-content: center;
+    gap: 25px;
+    flex-wrap: wrap;
+    padding: 0;
+
+    & > li {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        border-radius: 20px;
+        background-color: paleturquoise;
+        min-width: 125px;
+    }
+`;
+
+const CenterDiv = styled.div`
+    min-height: 120px;
+`;
+
+const ButtonWraper = styled.div`
+    display: flex;
+    gap: 20px;
+    padding: 14px 10px;
+`;
+
+interface ListProps {
+    items: Task[];
+    removeTask: (taskToRemove: Task) => void;
+}
+
+const List = (props: ListProps) => {
```

Comment:
you can destruct an object.
take a look at [MDN Destructuring Assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#:~:text=Similarly%2C%20you%20can%20destructure%20objects%20on%20the%20left%2Dhand%20side%20of%20the%20assignment.)

```suggestion
const List = ({ items, removeTask }: ListProps) => {
```

and after you can replace line 48
```JSX
const items = props.items;
```
with
```JSX
const items = items;
```